docs/en/get_started.md

```diff
@@ -37,30 +37,32 @@ We recommend that users follow our best practices to install MMDetection. Howeve
 
 ## Best Practices
 
-**Step 0.** Install [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
+**Step 0.** Install [MMEngine](https://github.com/open-mmlab/mmengine) and [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
 
 ```shell
 pip install -U openmim
-mim install mmcv-full
+mim install mmengine
+mim install mmcv-full>=2.0.0rc0
 ```
 
 **Step 1.** Install MMDetection.
 
 Case a: If you develop and run mmdet directly, install it from source:
 
 ```shell
-git clone https://github.com/open-mmlab/mmdetection.git
+git clone https://github.com/open-mmlab/mmdetection.git -b dev-3.x
+# "-b dev-3.x" means checkout to the `dev-3.x` branch.
 cd mmdetection
 pip install -v -e .
 # "-v" means verbose, or more output
 # "-e" means installing a project in editable mode,
 # thus any local modifications made to the code will take effect without reinstallation.
 ```
 
-Case b: If you use mmdet as a dependency or third-party package, install it with pip:
+Case b: If you use mmdet as a dependency or third-party package, install it with MIM:
 
 ```shell
-pip install mmdet
+mim install mmdet>=3.0.0rc0
```

Comment:
update line 90: 'Option (b). If you install mmdetection with pip'

Reply:
the demo is already updated